addons/auth_totp/views/res_users_views.xml

```diff
@@ -37,8 +37,10 @@
                                 Two-factor Authentication ("2FA") is a system of double authentication.
                                 The first one is done with your password and the second one with a code you get from a dedicated mobile app.
                                 Popular ones include Authy, Google Authenticator or the Microsoft Authenticator.
-                                <a href="https://www.odoo.com/documentation/16.0/applications/general/auth/2fa.html"
-                                   title="Learn More" target="_blank">Learn More</a>
+                                <a href="https://www.odoo.com/documentation/16.0/applications/general/auth/2fa.html" title="Learn More" target="_blank">
+                                    <i title="Documentation" class="fa fa-fw o_button_icon fa-info-circle"></i>
```

Comment:
I think that implies a new translation entry?

Meaning translations would need to be adapted.

If you have never done it, I can show you :)

Reply:
Hello @awa-odoo :) 

I tried something by generating a new pot file from the export translation settings and I copy pasted the file content into the `auth_totp.pot` file in the module. It created the 2 translations for the `Documentation` word but did nothing for the password placeholder though. I don't really think a translation is necessary for this one

Don't hesitate to tell me if it wasn't the right way to do so :) 
Thanks!

Reply:
Hello @amdi-odoo 

Almost perfect!
Just revert the few header lines you changed (we never change those, at least not that I've seen).

Thanks again and cheers!